contrib/terraform/vsphere/cloud-init.yaml

```diff
@@ -0,0 +1,37 @@
+---
+# cloud-config
+
+# sets the hostname to 'terraformed'
+hostname: terraformed
```

Comment:
Shall we remove this by default to allow the cloud provider to set the hostname? Alternatively, can we set it to something like the hostnames in `inventory.ini`?